### PR TITLE
Possible light/dark mode for the page :)

### PR DESCRIPTION
**Pull request by ivanosquis10**
Hi Midu, friends dev. I don't know if you like the idea but I still present it. I based it on the dark mode of another midu's project. Feel free to improve the code later if necessary 💯 

portfolio dev: https://github.com/midudev/porfolio.dev/blob/main/src/components/ThemeToggle.astro

light:
image

dark:
image

### CODE REVIEW

File: web/src/components/ToggleTheme.astro [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <MoonIcon
+      id="dark"
+      class="theme-toggle-icon absolute size-4 transition-all" />
+    <SystemIcon
+      id="system"
+      class="theme-toggle-icon absolute size-4 transition-all" />
+  </button>
+  <div
+    id="themes-menu"
+    class="absolute opacity-0 scale-80 top-8 right-0 text-sm p-1 min-w-[8rem] rounded-md border border-gray-100 bg-white/90 dark:bg-zinc-900/70 dark:border-gray-500/20 shadow-[0_3px_10px_rgb(0,0,0,0.2)] backdrop-blur-md z-50"
```
Comment (luis-cicada) [inline web/src/components/ToggleTheme.astro:25]: @ivanosquis10 I would recommend a `bg-zinc-900` as almost full black might not be best for dark mode.

**Review by luis-cicada — COMMENTED** [review]
I would recommend a `bg-zinc-900` | `bg-slate-900` as almost full black might not be best for dark mode.

File: web/src/components/ToggleTheme.astro (hunk 1 above)
Comment (ivanosquis10) [reply web/src/components/ToggleTheme.astro:25]: > @ivanosquis10 I would recommend a `bg-zinc-900` as almost full black might not be best for dark mode.

you say in the layout?

Comment (luis-cicada) [reply web/src/components/ToggleTheme.astro:25]: This one is fine..

File: web/src/layouts/Layout.astro [hunk 2]
```diff
@@ -26,17 +26,31 @@ const { title } = Astro.props
...
       >
       </div>
     </div>
     <slot />
   </body>
 </html>
-<style is:global></style>
+<style is:global>
+  :root {
+      color-scheme: light;
```
Comment (luis-cicada) [inline web/src/layouts/Layout.astro:44]: @ivanosquis10 we should add `color-scheme: light dark;`

File: web/src/layouts/Layout.astro [hunk 3]
```diff
@@ -26,17 +26,31 @@ const { title } = Astro.props
...
+    }
+
+    body {
+      color:rgba(0, 0, 0, 0.9)
```
Comment (luis-cicada) [inline web/src/layouts/Layout.astro:48]: @ivanosquis10 instead of full black something like `background: #212121;`

**Review by luis-cicada — COMMENTED** [review]
I added some comments

**Comment by ivanosquis10** [comment]
> I added some comments

nice, thanks for the help :)

**Pull request merged**